README.md

```diff
@@ -19,6 +19,20 @@ You can also see a list of missing data displayed as warnings in the developer c
 
 Check the [contributing](#contribute) section for more details.
 
+## Frequently asked questions
+
+**How do you define real time data?**  
+Real time data is defined as a data source that updates every 2 hours or better.  It should provide a breakdown by generation type.  Often fossil fuel generation (coal/gas/oil) is combined under a single heading like 'thermal' or 'conventional', this is not a problem.
+
+**Is non real time data useful?**  
```

Comment:
Maybe focus the title on the question of whether or not **delayed** data is still useful (instead of real-time)